ocs_ci/ocs/ui/acm_ui.py

```diff
@@ -181,6 +182,49 @@ def navigate_data_services(self):
             log.error("Couldn't navigate to data Services page on ACM UI")
             raise NoSuchElementException
 
+    def navigate_disaster_recovery(self):
+        """
+        Navigate to Disaster recovery page on ACM UI
+
+        """
+        log.info("Navigate to Disaster recovery page on ACM console")
+        find_element = self.wait_until_expected_text_is_found(
+            locator=self.acm_page_nav["data-services"],
+            expected_text="Data Services",
+            timeout=240,
+        )
+        if find_element:
```

Comment:
What will happen if element is not found?

Reply:
Else part ... Raises nosuchelementfound exception.